src/comm/TcpAcceptor.cc

```diff
@@ -419,6 +419,8 @@ Comm::TcpAcceptor::oldAccept(Comm::ConnectionPointer &details)
         }
     }
 
+    traceProtocol(5, "TCP Client REQUEST", conn, details);
```

Comment:
Please do not abuse the message-printing traceProtocols() to print accepted TCP connection details. If we were working at a TCP packet level, we could print SYN packets and such, of course, but this is not message dumping that traceProtocols() is dedicated to. It will also produce quite a bit of unwelcomed multiline noise where it is totally unnecessary. The line is also misplaced IMO, and polishing all this can be viewed as an out of scope change.


```suggestion
```